src/Microsoft.Framework.Runtime.Interfaces/IAssemblyLoadContext.cs

```diff
@@ -4,10 +4,31 @@
 
 namespace Microsoft.Framework.Runtime
 {
+    /// <summary>
+    /// A context in which assemblies can be loaded.
+    /// </summary>
     public interface IAssemblyLoadContext : IDisposable
     {
+        /// <summary>
+        /// Load an assembly by name.
+        /// </summary>
+        /// <param name="name">The name of the assembly.</param>
+        /// <returns>The loaded assembly.</returns>
```

Comment:
See I was looking around for examples of this and couldn't find anything consistent. Lets say you have a method that takes a type Foo as an argument:

```C#
/// <summary>
/// Makes a <see cref="Bar" /> from a <see cref="Foo" />.
/// </summary>
/// <param name="foo">The <see cref="Foo"> to transfrom.</param>
/// <returns> The transformed <see cref="Bar" />.</returns>
public Bar MakeFoo(Foo foo)
{

}
```

Not sure you need to repeat the cref in the param and returns...